course5/homework6/homework06_stdstr.cc

```diff
@@ -0,0 +1,104 @@
+#include <cstdio>
+#include <iostream>
+#include <cassert>
+#include <string>
+#include <cstring>
+
+void test_operators();
+void test_basics();
+
+
+void test_basics()
```

Comment:
This test_basic, and the test_basic in the other file are essentially the same. This call for a template! We'll discuss them today.